lib/rubocop/cop/style/mixin_grouping.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for grouping of mixins in `class` and `module` bodies.
+      # By default it enforces mixins to be placed in separate declarations,
+      # but it can be configured to enforce grouping them in one declaration.
+      #
+      # @example
+      #
+      #   EnforcedStyle: separated (default)
+      #
+      #   @bad
+      #   class Foo
+      #     include Bar, Qox
+      #   end
+      #
+      #   @good
+      #   class Foo
+      #     include Bar
+      #     include Qox
+      #   end
+      #
+      #   EnforcedStyle: grouped
+      #
+      #   @bad
+      #   class Foo
+      #     extend Bar
+      #     extend Qox
+      #   end
+      #
+      #   @good
+      #   class Foo
+      #     extend Bar, Qox
+      #   end
+      class MixinGrouping < Cop
+        include ConfigurableEnforcedStyle
+
+        MIXIN_METHODS = [:extend, :include, :prepend].freeze
+        MSG = 'Put mixins in %s.'.freeze
```

Comment:
I think the message is a bit confusing and should definitely mention `include` or `extend` for extra clarity.